altair/utils/headless.py

```diff
@@ -100,66 +83,73 @@ def connection_ok():
             .catch(function(err) { console.error(err); });
         """}
 
-def save_spec(spec, fp, mode=None, format=None, driver_timeout=10):
-    """Save a spec to file
+def spec_to_image_mimebundle(spec, format, mode,
+                             vega_version,
```

Comment:
Should this function have default vega/vegalite/vegaembed versions that match our generated APIs?